src/main/java/mods/eln/mechanical/SimpleShaft.kt

```diff
@@ -67,6 +71,24 @@ open class ShaftRender(entity: TransparentNodeEntity, desc: TransparentNodeDescr
     var connectionType: CableRenderType? = null
     open val cableRender: CableRenderDescriptor? = null
     var cableRefresh = true
+    // Sound:
+    private val soundLooper: ShaftSoundLooper?
```

Comment:
No, because it's only optionally initialized. I don't create it at all for shaft blocks that don't have sound.